mesonbuild/compilers.py

```diff
@@ -88,94 +88,94 @@ def is_library(fname):
                        }
 
-rust_buildtype_args = {'plain' : [],
-                       'debug' : ['-g'],
-                       'debugoptimized' : ['-g', '--opt-level', '2'],
-                       'release' : ['--opt-level', '3'],
-                       'minsize' : [],
+rust_buildtype_args = {'plain': [],
+                       'debug': ['-g'],
+                       'debugoptimized': ['-g', '--opt-level', '2'],
+                       'release': ['--opt-level', '3'],
+                       'minsize': [],
                        }
 
-d_gdc_buildtype_args = {'plain' : [],
-                       'debug' : ['-g', '-O0'],
-                       'debugoptimized' : ['-g', '-O'],
-                       'release' : ['-O3', '-frelease'],
-                       'minsize' : [],
+d_gdc_buildtype_args = {'plain': [],
+                        'debug': ['-g', '-O0'],
+                        'debugoptimized': ['-g', '-O'],
+                        'release': ['-O3', '-frelease'],
+                        'minsize': [],
                        }
 
-d_ldc_buildtype_args = {'plain' : [],
-                       'debug' : ['-g', '-O0'],
-                       'debugoptimized' : ['-g', '-O'],
-                       'release' : ['-O3', '-release'],
-                       'minsize' : [],
+d_ldc_buildtype_args = {'plain': [],
+                        'debug': ['-g', '-O0'],
+                        'debugoptimized': ['-g', '-O'],
+                        'release': ['-O3', '-release'],
+                        'minsize': [],
                        }
 
-d_dmd_buildtype_args = {'plain' : [],
-                       'debug' : ['-g'],
-                       'debugoptimized' : ['-g', '-O'],
-                       'release' : ['-O', '-release'],
-                       'minsize' : [],
+d_dmd_buildtype_args = {'plain': [],
+                        'debug': ['-g'],
+                        'debugoptimized': ['-g', '-O'],
+                        'release': ['-O', '-release'],
+                        'minsize': [],
                        }
 
-mono_buildtype_args = {'plain' : [],
-                       'debug' : ['-debug'],
+mono_buildtype_args = {'plain': [],
+                       'debug': ['-debug'],
                        'debugoptimized': ['-debug', '-optimize+'],
-                       'release' : ['-optimize+'],
-                       'minsize' : [],
-                       }
+                       'release': ['-optimize+'],
+                       'minsize': [],
+                      }
 
-swift_buildtype_args = {'plain' : [],
-                        'debug' : ['-g'],
+swift_buildtype_args = {'plain': [],
+                        'debug': ['-g'],
                         'debugoptimized': ['-g', '-O'],
-                        'release' : ['-O'],
-                        'minsize' : [],
-                        }
+                        'release': ['-O'],
+                        'minsize': [],
+                       }
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->